lib/fog/openstack/models/introspection/rules_collection.rb

```diff
@@ -0,0 +1,36 @@
+require 'fog/openstack/models/collection'
+require 'fog/openstack/models/introspection/rules'
+
+module Fog
+  module Introspection
+    class OpenStack
+      class RulesCollection < Fog::OpenStack::Collection
+        model Fog::Introspection::OpenStack::Rules
+
+        def all(_options = {})
+          load_response(service.list_rules, 'rules')
+        end
+
+        def summary(_options = {})
+          load_response(service.list_rules, 'rules')
```

Comment:
you can omit this, when it's the same as all. Summary delegates to all in the superclass